node/service/lib/registry.d.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import Device = require('./device');
+
+declare class Registry {
+    constructor(transport: Registry.Transport);
+    create(deviceInfo: { deviceId: string; }, done: Registry.ResponseCallback): void;
```

Comment:
`deviceInfo` can be a lot more than that. this just specifies that the `deviceId` is mandatory right? doesn't limit other properties in `deviceInfo`?